lib/vanagon/platform/windows.rb

```diff
@@ -75,7 +80,39 @@ def generate_nuget_package(project)
         "cp $(tempdir)/#{project.name}/#{project.name}-#{@architecture}.#{nuget_package_version(project.version, project.release)}.nupkg ./output/#{target_dir}/#{nuget_package_name(project)}"]
       end
 
-      # Method to derive the package name for the project.
+      # The specific bits used to generate a windows msi package for a given project
+      # [TBD - More to add]
+      #
+      # @param project [Vanagon::Project] project to build a msi package of
+      # @return [Array] list of commands required to build an msi package for the given project from a tarball
+      def generate_msi_package(project)
+        target_dir = project.repo ? output_dir(project.repo) : output_dir
+        ["mkdir -p output/#{target_dir}",
+        "mkdir -p $(tempdir)/STAGING",
+        # need to do things here.
+        "cp file-list $(tempdir)/#{project.name}/tools/file-list.txt", # May not need this (nuget only)
```

Comment:
we only need this if we need a list of files for the packaging process. This file looks something like:
```
C:/Program Files/Puppet Labs
C:/Program Files/Puppet Labs/Puppet
C:/Program Files/Puppet Labs/Puppet/bin
C:/Program Files/Puppet Labs/Puppet/bin/ruby.exe
C:/ProgramData/PuppetLabs
```
It includes a listing of files, directories, and empty directories.